cmake/draco.cmake

```diff
@@ -0,0 +1,20 @@
+#
+option(WITH_DRACO "Choose if Draco support should be built" TRUE)
+find_package(Draco QUIET 1.3.6)
```

Comment:
We should set this to EXACT match for the moment. Once we get things working, we're going to have to go to the Draco team to fix up their CMake layout for 1.4.x